{"labels": [{"value": "Login_related_test", "name": "suite"}, {"value": "TestSendR", "name": "subSuite"}, {"value": "testcases", "name": "parentSuite"}, {"value": "DESKTOP-BQ40Q36", "name": "host"}, {"value": "89148-MainThread", "name": "thread"}, {"value": "pytest", "name": "framework"}, {"value": "cpython3", "name": "language"}, {"value": "testcases.Login_related_test", "name": "package"}], "name": "test_read_message", "fullName": "testcases.Login_related_test.TestSendR#test_read_message", "testCaseId": "8f68eeef87020c50cb51fdd7d2d71ca4", "start": 1638519559137, "stop": 1638519559139, "statusDetails": {"message": "AttributeError: 'str' object has no attribute 'items'", "trace": "self = <testcases.Login_related_test.TestSendR object at 0x0000023001C15160>\n\n    def test_read_message(self):\n        token = yamlUtil().read_relation_yaml()\n        url = \"http://172.16.11.40:8082/app/readMsg\"\n        # yamlUtil().write_relation_yaml(url)\n        data = {\"msgId\": 247}\n>       rep = requests.request(\"POST\", url=url, data=data, headers=token)\n\nLogin_related_test.py:49: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nC:\\Python\\lib\\site-packages\\requests\\api.py:61: in request\n    return session.request(method=method, url=url, **kwargs)\nC:\\Python\\lib\\site-packages\\requests\\sessions.py:516: in request\n    prep = self.prepare_request(req)\nC:\\Python\\lib\\site-packages\\requests\\sessions.py:459: in prepare_request\n    hooks=merge_hooks(request.hooks, self.hooks),\nC:\\Python\\lib\\site-packages\\requests\\models.py:315: in prepare\n    self.prepare_headers(headers)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <PreparedRequest [POST]>, headers = 'ghFile/1638519557170/1.jpg'\n\n    def prepare_headers(self, headers):\n        \"\"\"Prepares the given HTTP headers.\"\"\"\n    \n        self.headers = CaseInsensitiveDict()\n        if headers:\n>           for header in headers.items():\nE           AttributeError: 'str' object has no attribute 'items'\n\nC:\\Python\\lib\\site-packages\\requests\\models.py:447: AttributeError"}, "status": "broken", "uuid": "28f5b3a5-94be-4999-baca-507adf8ea31c", "historyId": "ef9d32d79632d38baddce6e0b7ab0a15"}